wiki/Storyboarding/en.md

```diff
@@ -6,6 +6,6 @@
 
 ## Creating a storyboard
 
-The [Storyboard Editor](/wiki/Storyboard_Editor) (SBE) is located in the Design Tab (keyboard shortcut F2) in the Beatmap Editor. Fundamentally, the SBE offers users an easy way to control the movement of simple sprites, but is often regarded as clunky and unusable for anything beyond the most basic of storyboards. If one would wish to create a widescreen storyboard, you can do so by going to the song setup dialog.
+The [Storyboard Editor](/wiki/Storyboard_Editor) (SBE) is located in the Design Tab (keyboard shortcut F2) in the Beatmap Editor. Fundamentally, the SBE offers users an easy way to control the movement of sprites, but is often regarded as clunky and unusable for anything beyond the basics of storyboarding. If one would wish to create a widescreen storyboard, you can do so by going to the song setup dialog, then go to the last page, then check the box that says 'Widescreen Support'.
 
-Users skilled in storyboard creation often use the backend code created by the SBE, entitled [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS) instead, as it affords them more control. See that article for details.
+Users skilled in storyboard creation often use the backend code created by the SBE, entitled [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS) instead, as it offers them more control. See the article for details.
```

Comment:
The article change feels more confusing:

> See the article for details.

*Which* article? The previous version avoided that by explicitly referring to the previous sentence.

Reply:
I reverted the change